apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/PgTests.cs

```diff
@@ -48,6 +48,12 @@ public class PgTests
     [Fact]
     public async Task BindPgNumericEmptyArray() =>
         await TestBindNonNull(SpannerDbType.ArrayOf(SpannerDbType.PgNumeric), new PgNumeric[] { });
+    
+    [Fact]
+    public async Task BindPgJson() => await TestBindNonNull(
+            SpannerDbType.PgJsonb,
```

Comment:
I think this should be outdented so that it's under the "i" of "public". (Or just put it all on one line, possibly breaking after `=>`.

Reply:
Done.